docs/tutorials/timeseries/forecasting-faq.md

```diff
@@ -1,42 +1,40 @@
 # AutoGluon Time Series FAQ
 
+## What forecasting tasks can AutoGluon be used for?
+AutoGluon can generate **probabilistic** multi-step-ahead forecasts for one or multiple **univariate** time series.
+For example, you can use AutoGluon to forecast daily sales of multiple products over the next month.
 
-## Where can I find more information about the models/metrics?
+AutoGluon also supports additional information, such as time-independent static features (e.g., location of the store)
+and time-dependent covariates (e.g., price of the product each day).
+See the [In Depth Tutorial](forecasting-indepth.ipynb) for more details.
 
-Metrics are implemented in the `autogluon.timeseries.evaluator` module. We also follow some of
-the same conventions followed by GluonTS in their evaluation.
-Please refer to
-the GluonTS [documentation](https://ts.gluon.ai/stable/api/gluonts/gluonts.html) and
-[github](https://github.com/awslabs/gluon-ts) for further information.
+Currently, AutoGluon does not support features such as hierarchical forecasting and forecast explainability, but we will consider adding them in the future.
```

Comment:
Is there an issue with tracking feature requests related to hierarchical forecasting and explainability from users? If yes, we can remove the "but" statement.

Reply:
Just to clarify, you think we should open an issue for each of these features on GitHub & link them here so that users can "+1" if they care about these?

Reply:
I support adding feature issues to our backlog and linking them to release milestones to ensure we stay on track. Additionally, if users have already requested certain features, we can include that information in the issue description.